src/aleph/vm/orchestrator/status.py

```diff
@@ -118,10 +118,15 @@ async def check_ipv6(session: ClientSession, vm_id: ItemHash) -> bool:
 async def check_internet(session: ClientSession, vm_id: ItemHash) -> bool:
     """Check that the VM has internet connectivity. This requires DNS, IP, HTTP and TLS to work."""
     try:
-        result: dict = await get_json_from_vm(session, vm_id, "/internet")
-        assert result["result"] == HTTPOk.status_code
-        assert "Server" in result["headers"]
-        return True
+        response: dict = await get_json_from_vm(session, vm_id, "/internet")
+
+        # The HTTP Header "Server" must always be present in the result.
+        if "Server" not in response["headers"]:
```

Comment:
it's present event if there is no internet?

I'm not surr I see the value of this check, especially if we dont do anything with it.. Maybe this should be moved in a test instead?

Reply:
Yes, it is added by the orchestrator/proxy.